src/pudl/constants.py

```diff
@@ -2586,6 +2640,8 @@
         'ferc_small_power_producer_docket_no': pd.StringDtype(),
         'fluidized_bed_tech': pd.BooleanDtype(),
         'fraction_owned': float,
+        # Added by AES for NM table (might want to consider merging with another fuel label)
+        'fuel_class': pd.CategoricalDtype(categories=['pv', 'wind', 'chpcogen', 'other', 'total']),
```

Comment:
I agree that it might be good to make this column consistent with the other fuel labels. possibly a `fuel_type_code*`. This may be a good thing to assess and move towards once we have run all of the tables through this transform phase 1 step.

Reply:
would this be the same `fuel_type_code` that governed the mcoe table breakdown? Just checking.

Reply:
yes, exactly!

Reply:
Cool - I made an issue about this because I think it'll require a little more coordination / discussion to complete!